### PR TITLE
add sidebar/nav and routes Co-authored-by: Amr Ahmed

### LINKED ISSUE #2: Navigation Sidebar / Panel - Web 1pt
### Create the App Navigation
- [x] Add a navigation panel
- [x] The navigation panel should have the following button links: 👇 
   - Doctors.
   - "Reserve" form.
   - My reservations.
   - "Add Doctor - the link is visible to everybody.
   - Delete Doctor - the link is visible to everybody.

### PR DESCRIPTION
**Pull request by codepantha**
## Changes introduced via this PR: 👇 

- [x] Add a navigation panel with links
- [x] Add routes

### CODE REVIEW

**Review by AlexRS90 — APPROVED** [review]
# Hello @codepantherr
###What an amazing job, it's working as expected, the styling is like in the design provided and also you created functional components for other features in the project, good one!!

## There is nothing else to say other than... is time to merge!!! 🎸  🕺  🎸  🕺

File: src/components/sidebar/Sidebar.jsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        <li className="nav-item">
+          <NavLink to="/" className="nav-link">
+            Doctors
+          </NavLink>
+        </li>
+        <li className="nav-item">
+          <NavLink to="/book-appointment" className="nav-link">
+            Reserve
+          </NavLink>
+        </li>
```
Comment (AlexRS90) [inline src/components/sidebar/Sidebar.jsx]: - [ ] This is an optional Change:

Everything is working as expected but I believe is a good idea to improve de UX to close the menu after the user selects an option.

**Review by AlexRS90 — APPROVED** [review]
## Hello @Amrhub 
It's amazing how you implemented the dark them, taking care of all details like the logo in the navbar and the project is also responsive!! 
## Congratulations!! 
## Feel free to merge it! 🚀  🕺  🎸

File: src/components/sidebar/Sidebar.jsx (hunk 1 above)
Comment (Amrhub) [reply src/components/sidebar/Sidebar.jsx]: You're right, I will work on that now 🤩

File: src/components/ThemeToggler/ThemeToggler.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+const ThemeToggler = () => {
+  const [theme, setTheme] = useState('');
+
+  useEffect(() => {
+    setTheme(localStorage.getItem('theme'));
+    const localTheme = theme;
+    if (localTheme === 'dark') {
+      document.getElementById('theme-toggler').checked = true;
+    }
+  }, [theme]);
```
Comment (AlexRS90) [inline src/components/ThemeToggler/ThemeToggler.jsx:12]: This is a great feature, congratulations!! 

keep rocking!! 🎸  🕺

**Review by AlexRS90 — APPROVED** [review]
## Hello @Amrhub 
Your project is complete!!
There is nothing else to say other than... it's time to merge!!! 🚀  🟢

**Comment by Amrhub** [comment]
@AlexRS90 Thank you for the review ❤️

**Pull request merged**